Library/Homebrew/cask/cmd/create.rb

```diff
@@ -30,6 +30,7 @@ def self.template(cask_token)
 
             url "https://"
             name ''
```

Comment:
Please update the quotes here to double quotes while you're at it.

Reply:
Done.